### ISSUE TITLE
Tree View methods does not work on Windows 2012 R2 (64-bit)

### ISSUE THREAD

**Issue by jagadeesh1983** [pywinauto/pywinauto#371]
Reference: #93 Tree Methods Don't Work?
Hi,

I am using Python 2.7 (64 bit) with pyWinAuto 0.4 (64 bit) on Windows 2012 R2 (64 bit) to automate 64-bit application.
I am getting following error while accessing Tree Method ie GetItem() to read TreeView items. 
ctypes.ArgumentError: argument 4: <type 'exceptions.OverflowError'>: long int too long to convert

Here is my Code sample:
```python
from pywinauto import application
import time
from pywinauto import tests
from pywinauto.findbestmatch import MatchError
from pywinauto import findwindows
from pywinauto.timings import Timings

app = application.Application()
app.start("I:\\PT857\\ORACLE\\pt857-101-r1-prj-stingray\\bin\\client\\winx86\\pside.exe -CT ORACLE -CD T57123N5 -CO QEDMO -CP QEDMO")
time.sleep(5)
app.window_(title_re = 'Application Designer - *').MenuSelect('File->New')
app.NewDefinition.ListBox.Select('Application Package')
time.sleep(1)
app.NewDefinition.OK.ClickInput()
app.window_(title_re = 'Application Designer - *').MenuSelect('Insert->Package')
app.Definition.PackageNameEdit.SetEditText('SubPackageB')
app.Definition.TypeKeys('{ENTER}')
app.window_(title_re='.*Application Designer.*').TreeView.GetItem('\(unnamed)\SubPackageB').State()
```

Output:
```
Traceback (most recent call last):
  File "C:\Users\jkanthar\workspace\TEST123\TreeView\AppPackage.py", line 18, in <module>
    app.window_(title_re='.*Application Designer.*').TreeView.GetItem('\(unnamed)\SubPackageB').State()
  File "C:\Python27\lib\site-packages\pywinauto\controls\common_controls.py", line 965, in GetItem
    texts = [r.Text() for r in roots]
  File "C:\Python27\lib\site-packages\pywinauto\controls\common_controls.py", line 549, in Text
    return self._readitem()[1]
  File "C:\Python27\lib\site-packages\pywinauto\controls\common_controls.py", line 839, in _readitem
    remote_mem)
ctypes.ArgumentError: argument 4: <type 'exceptions.OverflowError'>: long int too long to convert
```

However, same code is working fine on Windows 7 (64 bit) with Python 2.7 (64 bit) & pyWinAuto 0.4(64 bit).

I have observed that item.pszText under _readitem() function is assigned a 64-bit Pointer in Windows 2012 R2 (64-bit) which is causing Overflow error when the entry is read through retval variable under _readitem().
Whereas in Windows 7 (64-bit), item.pszText under _readitem() function is assigned 32-bit Pointer due to which above code works in Windows 7 (64-bit).

Please help how to fix this issue in Windows 2012 R2 (64-bit).

Thanks,
Jagadeesh

**Comment by vasily-v-ryabov** [comment]
Hi @jagadeesh1983 this is already reported on StackOverflow: https://stackoverflow.com/q/44210866/3648361
The fix is on the way and will be included into coming 0.6.3.

**Comment by jagadeesh1983** [comment]
Hi Vasily,

Any idea when is pywinauto 0.6.3 (64-bit) available for customers.

Thanks,
Jagadeesh

**Comment by vasily-v-ryabov** [comment]
Hope it will be published this week or next one. I'll let you know when master branch will contain the fix.

**Comment by jagadeesh1983** [comment]
Sure. Thanks for the update.

**Comment by vasily-v-ryabov** [comment]
See PR #371.

**Comment by jagadeesh1983** [comment]
Hi Vasily,

Could you please confirm this new pywinauto 0.6.3 (64-bit) version supports Python 2.7.4 (x64) and 3.6.0 (x64) versions.

Thanks,
Jagadeesh

**Comment by jagadeesh1983** [comment]
Hi Vasily,

Could you please confirm this new pywinauto 0.6.3 (64-bit) version supports
Python 2.7.4 (x64) and 3.6.0 (x64) versions.

Thanks,
Jagadeesh

**Comment by vasily-v-ryabov** [comment]
Hi @jagadeesh1983 we auto-test every commit and every pull request on AppVeyor with [the following Python versions](https://www.appveyor.com/docs/build-environment/#python):
```
Python 2.7.13 x86 (C:\Python27 - default in PATH)
Python 2.7.13 x64 (C:\Python27-x64)
Python 3.3.5 x86 (C:\Python33)
Python 3.3.5 x64 (C:\Python33-x64)
Python 3.4.4 x86 (C:\Python34)
Python 3.4.4 x64 (C:\Python34-x64)
Python 3.5.3 x86 (C:\Python35)
Python 3.5.3 x64 (C:\Python35-x64)
Python 3.6.1 x86 (C:\Python36)
Python 3.6.1 x64 (C:\Python36-x64) (some time ago it was 3.6.0 and it worked also)
```

On my local machine I use Python 2.7.8. So assume it should work on Python 2.7.4 as well. And even on Python 2.6 (though officially we don't support it).

**Comment by vasily-v-ryabov** [comment]
Can you confirm this `ctypes.ArgumentError` has gone in 0.6.3?

**Comment by jagadeesh1983** [comment]
Hi Vasily,

Thanks for the confirmation.

Yes, ctypes.ArgumentError got resolved with new pywinauto 0.6.3 (64-bit) version.

I have used following combination and both worked fine.
1. Python 3.6.0 (x64) + pyWinAuto 0.6.3 (x64) on Windows 2012 R2 (x64) 
2. Python 2.7.4 (x64) + pyWinAuto 0.6.3 (x64) on Windows 2012 R2 (x64) 

Thanks,
Jagadeesh
